uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerAuthenticatorUpdatedVersionImpl.java

```diff
@@ -85,15 +85,10 @@ public void performDeviceRegistration(@NonNull final String username,
                         unRegisterBtn.exists()
                 );
 
-                Assert.assertTrue(
-                        "Microsoft Authenticator - Unregister Button is clickable.",
-                        unRegisterBtn.isClickable()
-                );
-
                 // after device registration, make sure that the current registration upn matches with
                 // with what was passed in
                 final UiObject currentRegistration = UiAutomatorUtils.obtainUiObjectWithResourceId(
-                        "com.azure.authenticator:id/current_registered_email"
+                        "workPlaceRegisteredAccountEmailText"
```

Comment:
So this resource id got changed?

Reply:
From what I saw, yes. This code is only being ran in one test case though.